akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteGatePiercingSpec.scala

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.remote
+
+import akka.remote.transport.AssociationHandle
+
+import language.postfixOps
+import scala.concurrent.duration._
+import com.typesafe.config.ConfigFactory
+import akka.actor._
+import akka.remote.testconductor.RoleName
+import akka.remote.transport.ThrottlerTransportAdapter.{ ForceDisassociateExplicitly, ForceDisassociate, Direction }
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.remote.testkit.STMultiNodeSpec
+import akka.testkit._
+import akka.actor.ActorIdentity
+import akka.remote.testconductor.RoleName
+import akka.actor.Identify
+import scala.concurrent.Await
+
+object RemoteGatePiercingSpec extends MultiNodeConfig {
+  val first = role("first")
+  val second = role("second")
+
+  commonConfig(debugConfig(on = false).withFallback(
+    ConfigFactory.parseString("""
+      akka.loglevel = INFO
+      akka.remote.log-remote-lifecycle-events = INFO
+      akka.remote.transport-failure-detector.acceptable-heartbeat-pause = 5 s
+                              """)))
+
+  nodeConfig(first)(
+    ConfigFactory.parseString("akka.remote.retry-gate-closed-for  = 1 d # Keep it long"))
+
+  nodeConfig(second)(
+    ConfigFactory.parseString("akka.remote.retry-gate-closed-for  = 1 s # Keep it short"))
+
+  testTransport(on = true)
+
+  class Subject extends Actor {
+    def receive = {
+      case "shutdown" ⇒ context.system.terminate()
+      case msg        ⇒ sender() ! msg
```

Comment:
This test is not sending these messages to this actor. Remove them to reduce noise.